src/Router/src/Target/Namespaced.php

```diff
@@ -30,16 +30,12 @@ final class Namespaced extends AbstractTarget
     private $inflector;
 
     /**
-     * @param string $namespace
-     * @param string $postfix
-     * @param int    $options
      * @param string $defaultAction
```

Comment:
I think. we have to get rid of this line

Reply:
removed https://github.com/spiral/framework/pull/472/commits/6818a95bda8decc5f5579873eb9c98cd2ebfcb83